src/processor/prefix-external-id.spec.ts

```diff
@@ -79,20 +81,33 @@ describe('PrefixExternalId', () => {
           'documents-3',
         ),
       ],
+      articleLookupTable: new Map<string, ExternalLink>([
+        ['key1', { externalDocumentId: 'article-external-id-1' }],
+        ['key2', { externalDocumentId: 'article-external-id-2' }],
+        ['key3', { externalDocumentId: 'article-external-id-3' }],
+      ]),
     };
 
     const result = await prefixExternalIdProcessor.run(content);
 
     (['labels', 'categories', 'documents'] as (keyof typeof result)[]).forEach(
       (entityType) => {
-        expect(result[entityType].length).toBe(3);
-        result[entityType].forEach((item, index) =>
+        const entities = result[entityType] as ExternalIdentifiable[];
+        expect(entities.length).toBe(3);
+        entities.forEach((item, index) =>
           expect(item.externalId).toBe(
             `${EXTERNAL_ID_PREFIX}${entityType}-${index + 1}`,
           ),
         );
       },
     );
+
+    const entriesArray = Array.from(result.articleLookupTable?.entries() ?? []);
+    entriesArray.forEach(([key, externalLink], index) =>
```

Comment:
Here you should also assert that the array has any items in it. Without that you may verify nothing.